examples/me/client/package.json

```diff
@@ -66,7 +66,7 @@
     "html-webpack-plugin": "^4.3.0",
     "webpack": "^5.53.0",
     "webpack-cli": "^4.8.0",
-    "webpack-dev-server": "^3.11.0",
+    "webpack-dev-server": "^4.11.1",
```

Comment:
Is this a breaking change in the webpack dev server?

Reply:
yes, but it is isolated just to the examples folder I think it wouldn't be a high impact on our SDK. WDYT?